README.md

```diff
@@ -1,596 +1,493 @@
-
-#### Below is an overview of the different steps in this file:
-
-1. [Basic setup](#basic-setup)
-2. [Setting up the developer environment](#setting-up-the-developer-environment)
-3. [Creating an OpenST object](#creating-an-openst-object)
-4. [Adding accounts](#adding-accounts)
-5. [Deploying an EIP20 contract](#deploying-an-eip20-contract)
-6. [Economy setup](#economy-setup)
-7. [User setup](#user-setup)
-8. [Balance verification: Before execute rule](#balance-verification-before-execute-rule)
-9. [Execute sample rule](#execute-sample-rule)
-10. [Balance verification: After execute rule](#balance-verification-after-execute-rule)
-
-#### Basic setup
-
-1.  Install the following packages
-
-```bash
-sudo apt-get update
-sudo apt-get install nodejs
-sudo apt-get install npm
-sudo apt-get install software-properties-common
-```
-
-2.  Install OpenST.js in your project using npm
+##  Setup
+This library assumes that nodejs and geth are installed and running. To install OpenST.js in your project using npm:
 
 ```bash
 $ npm install @openstfoundation/openst.js --save
-
-
-```
-3.  Install Geth
-
-    This code was tested with geth version: 1.7.3-stable. Other higher versions should also work.
-
-```bash
-curl https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.7.3-4bb3c89d.tar.gz | tar xvz
-mv geth-linux-amd64-1.7.3-4bb3c89d /usr/local/bin
-ln -s /usr/local/bin/geth-linux-amd64-1.7.3-4bb3c89d/geth /usr/local/bin/geth
-export PATH="$PATH:/usr/local/bin/geth-linux-amd64-1.7.3-4bb3c89d"
 ```
 
-4.  Sync your development machine with one of the following test environments
-
-    - https://ropsten.etherscan.io/
-    - https://kovan.etherscan.io/
-    - https://rinkeby.etherscan.io/
-
-5.  Create the following addresses and fund them with gas
-
-    - deployerAddress - address that deploys the EIP20Token, TokenRules, TokenHolder, and rule contracts
-    - organizationAddress - address that registers rule contracts in the TokenRules contract
-    - wallet1 - owner1 of the TokenHolder contract
-    - wallet2 - owner2 of the TokenHolder contract
-    - ephemeralKey - the key that will be authorized by owners in the TokenHolder contract; this key will sign execute rule transactions
-    - facilitatorAddress - the address that will facilitate execute rule transactions
-
-#### Setting up the developer environment
-
-###### Initialize the chain on developer machine
-
-The below command creates a json file (`~/openst-setup/config.json`) with all the needed addresses and other constants. It also starts Geth process for the chain.
-
-This is a quick-start option. You could also choose to do this step manually.
-
-```bash
-node ./node_modules/\@openstfoundation/openst.js/tools/initDevEnv.js ~
-
-
-```
-
-#### Creating an OpenST object
-OpenST.js is a thin layer over web3.js. Its constructor arguments are same as that of web3.js.
+This code was tested with geth version: 1.7.3-stable. Other higher versions should also work.
```

Comment:
this needs to be updated; we re testing with geth 1.8.23

Reply:
granted, we re not using any of the new EIPs; perhaps just remove this line; state that it works for Ethereum Byzantium and Petersburg